jp/components/drop_down.md

```diff
@@ -0,0 +1,283 @@
+﻿---
+title: Drop Down コンポーネント
+_description: Ignite UI for Angular Drop Down は、単一項目の選択が可能なグループ化された項目のリストにスクロールを表示します。
+_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Angular Drop Down components, Angular Drop Down controls
+---
+
+##Drop Down
+<p class="highlight">Ignite UI for Angular Drop Down は、単一項目の選択が可能なグループ項目をスクロールできるリストで表示します。項目をクリックまたはタップして選択し、ドロップダウンを閉じます。</p>
+<div class="divider"></div>
+
+### Drop Down デモ
+<div class="sample-container" style="height:240px">
+    <iframe id="dropdown-sample-3-iframe" src='{environment:demosBaseUrl}/dropdown-sample-3' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+<button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="dropdown-sample-3-iframe" data-demos-base-url="{environment:demosBaseUrl}">stackblitz で開く</button>
+</div>
+<div class="divider--half"></div>
+
+> [!NOTE]
+> Ignite UI for Angular コンポーネントをプロジェクトに追加する前に、すべての必要な依存関係を構成し、プロジェクトのセットアップが正しく完了したことを確認してください。[**インストール**](https://jp.infragistics.com/products/ignite-ui-angular/getting-started#installation) のトピックで手順を確認できます。
+
+###使用方法
+Ignite UI for Angular Drop Down コンポーネントを初期化する前に **IgxDropDownModule** を **app.module.ts** ファイルにインポートします。
+
+```typescript
+// app.module.ts
+
+...
+import { IgxDropDownModule } from 'igniteui-angular/main';
+
+@NgModule({
+    ...
+    imports: [..., IgxDropDownModule],
+    ...
+})
+export class AppModule {}
+```
+
+選択可能な複数のオプション項目を提供するシンプルなドロップダウンを作成します。**igx-drop-down** コンポーネントと **igx-toggle** を使用してドロップダウンの開く/閉じるを有効にします。
+
+次にすべての必要なモジュールを含み、**app.module.ts** ファイルにインポートします。
+
+```typescript
+// app.module.ts
+
+...
+import {
+    IgxDropDownModule,
+    IgxToggleModule
+} from 'igniteui-angular/main';
+
+@NgModule({
+    ...
+    imports: [..., IgxDropDownModule, IgxToggleModule],
+})
+export class AppModule {}
+```　
+構成の後、すべてのオプションを  [**IgxDropDownComponent**](https://www.infragistics.com/products/ignite-ui-angular/angular/components/drop_down.html) コンポーネントに表示します。ドロップダウンは、ドロップダウン項目として複数のオプションを含みます。
```

Comment:
Please update the URL.  It can actually be changed to a relative URL like `drop_down.html`.  I forget if they went and changed all of them to `.html` or `.md`, so please reference the other topics to see how it's done.  Otherwise, this will always go to the EN production site.  Even if you just change it to the JP site, then it won't work on staging.

Reply:
Updated '.md'.